bin/setup

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
There's already a script called `first_install ` in scripts directory that do something like that, also this has nothing to do with rails 4.2 update ;-)